src/app/find/errorReports/ErrorReportInformationSelect.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Header from '../../../components/header';
+import Button from '../../../components/button';
+import Checkbox from '../../../components/checkbox';
+
+const ErrorReportInformationSelect = (props) => {
+  const {
+    locationData,
+    errorReportServices,
+    generalLocationError,
+    onGeneralLocationChange,
+    onServiceChange,
+    goToViewLocation,
+  } = props;
+
+  return (
+    <div>
+      <Header size="large" className="locationTitle">Error Report</Header>
+      <div className="mx-4 mb-3 text-left">
+        <div className="w-100 mr-5">
+          <Header size="medium" className="mb-3">
+            Which parts of the information has errors?
+          </Header>
+        </div>
+        <div>
+          <div>
+            <Checkbox
+              key="generalLocationError"
+              name="generalLocationError"
+              label="Information about the location"
+              checked={generalLocationError}
+              onChange={onGeneralLocationChange}
+            />
+          </div>
+          {locationData.Services.map(service => (
+            <div key={service.id}>
+              <Checkbox
+                key={service.id}
+                name={service.id}
+                label={service.name}
+                checked={errorReportServices.includes(service.id)}
+                onChange={() => onServiceChange(service.id)}
+              />
+            </div>
+          ))}
+        </div>
+      </div>
+
+      <div className="mx-4 mb-1">
+        <Link to={`${props.match.url}/text`}>
```

Comment:
Hmm, so despite the huge amount of logic concentrated in in the container, the URL schema, i.e. the high-level flow of the pages, is actually the one thing that _isn't_?

I'd try to at least keep that encapsulated there.
Just means passing a function prop to this component, and then invoking it on the button, instead of wrapping it in a Link (!?) and making it do nothing.

Reply:
Yeah, that's the original way I had it, however I was running into issues where the URL would update, but not render the new Route.

Reply:
Originally all navigation was like the `goToViewLocation` function in `mapDispatchToProps` in the `ErrorReportContainer.jsx` file.

Reply:
I'll take another look at what I might have been doing wrong previously.